### PR TITLE
Update readme template to match core-plans requirements

### PR DESCRIPTION
**Pull request by predominant**
Signed-off-by: Graham Weldon <graham@grahamweldon.com>

tenor-158560484

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: components/hab/static/template_README.md [hunk 1]
```diff
@@ -1,9 +1,96 @@
...
+A binary package is something that packages up a standalone binary, something that does not need to run under the Habitat supervisor (i.e. core/dep). They are often used as dependencies of other packages. Binary packages do not have a run file and do not need to define pkg_svc_run in the plan.sh file.
 
 ## Usage
 
-Describe the general usage for the `{{ pkg_name }}` plan
+How would a user use this package?  i.e. can a user simply call the package as a dependency of their application?  Or is there more they need to do?
+
+## Bindings
+
+*(This is only required for service packages, not [binary wrapper packages](https://www.habitat.sh/docs/best-practices/#binary-wrapper-packages)))*
```
Comment (baumanj) [inline components/hab/static/template_README.md]: Mismatched parenthesis

File: components/hab/static/template_README.md [hunk 2]
```diff
@@ -1,9 +1,96 @@
...
+
+Check out [the Habitat docs on standalone](https://www.habitat.sh/docs/using-habitat/#standalone) for more details on what the standalone topology is and what it does.
+
+If this plan can be used with the standalone topology, how do you do it?
+
+Checkout [the core/postgresql](https://github.com/habitat-sh/core-plans/tree/master/postgresql) README for a good example of this.
+
+### Leader-Follower
+
+*(This is only required for service packages, not [binary wrapper packages](https://www.habitat.sh/docs/best-practices/#binary-wrapper-packages)))*
```
Comment (baumanj) [inline components/hab/static/template_README.md]: Mismatched parenthesis

File: components/hab/static/template_README.md [hunk 3]
```diff
@@ -1,9 +1,96 @@
...
+
+If this plan can be used with the leader/follower topology, how do you do it?
+
+Check out [the Habitat docs on Leader-Follower](https://www.habitat.sh/docs/using-habitat/#leader-follower-topology) for more details on what the leader-follower topology is and what it does.
+
+Checkout [the core/postgresql](https://github.com/habitat-sh/core-plans/tree/master/postgresql) README for a good example of this (look under the Clustering heading)
+
+## Update Strategies
+
+*(This is only required for service packages, not [binary wrapper packages](https://www.habitat.sh/docs/best-practices/#binary-wrapper-packages)))*
```
Comment (baumanj) [inline components/hab/static/template_README.md]: Mismatched parenthesis

File: components/hab/static/template_README.md [hunk 4]
```diff
@@ -1,9 +1,96 @@
...
+
+What update strategies would work best for this plan?
+
+Checkout [the update strategy documentation](https://www.habitat.sh/docs/using-habitat/#update-strategy) for information on the strategies Habitat supports.
+
+### Configuration Updates
+
+*(This is only required for service packages, not [binary wrapper packages](https://www.habitat.sh/docs/best-practices/#binary-wrapper-packages)))*
```
Comment (baumanj) [inline components/hab/static/template_README.md]: Mismatched parenthesis

File: components/hab/static/template_README.md [hunk 5]
```diff
@@ -1,9 +1,96 @@
...
+
+Checkout the [configuration update](https://www.habitat.sh/docs/using-habitat/#configuration-updates) documentation for more information on what configuration updates are and how they are executed.
+
+Link to the plan's default.toml file to list all the configurable values of the plan.
+
+If your plan has configuration values that require a complete rebuild when updated, note those here.
+
+## Scaling
+*(This is only required for service packages, not [binary wrapper packages](https://www.habitat.sh/docs/best-practices/#binary-wrapper-packages)))*
```
Comment (baumanj) [inline components/hab/static/template_README.md]: Mismatched parenthesis

File: components/hab/static/template_README.md [hunk 6]
```diff
@@ -1,9 +1,96 @@
...
+
+(Optional, but recommended)
+
+How would a user scale this service?
+
+Can this service be run in a cluster and/or as high availability?
+
+## Monitoring
+
+*(This is only required for service packages, not [binary wrapper packages](https://www.habitat.sh/docs/best-practices/#binary-wrapper-packages)))*
```
Comment (baumanj) [inline components/hab/static/template_README.md]: Mismatched parenthesis

**Comment by baumanj** [comment]
I'm not sure I understand what's going on here @predominant. Can you walk me through the workflow where we want this to live in the `habitat` repo? I get that it was here before and you're just updating it, but I think maybe we should instead be removing it.

**Comment by predominant** [comment]
@baumanj 

* User creates a habitat plan: `hab plan init`
* Readme placeholder contains all the information they need

This is better, in my opinion than:

* User creates a habitat plan: `hab plan init`
* Readme contains link to get template
* User copies template
* User pastes template

**Comment by baumanj** [comment]
Oh, I see. Thanks for the explanation @predominant. Perhaps it's [`core-plans`'s version](https://github.com/habitat-sh/core-plans/blob/master/README_TEMPLATE_FOR_PLANS.md) that should link _here_. I'll look into that.

**Pull request merged**